workshops/views.py

```diff
@@ -1847,7 +1847,7 @@ def all_trainings(request):
         .values('event').annotate(finished=Count('person'))
     finished = {f['event']: f['finished'] for f in finished}
 
-    trainings = Task.objects.filter(role=learner).filter(event__tags=ttt).order_by('event__start') \
+    trainings = Task.objects.filter(role=learner).filter(event__tags=ttt).order_by('-event__start') \
```

Comment:
Not sure if Greg wanted to change the only sorting way or to have an option to choose sorting. @gvwilson ?